tests/test_gaudi_configuration.py

```diff
@@ -13,35 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import filecmp
-import tempfile
 import unittest
-from pathlib import Path
-from typing import List
 
 from optimum.habana import GaudiConfig
 
 
-BF16_OPS_REFERENCE_FILE = Path(__file__).parent.resolve() / Path("configs/bf16_ops.txt")
-FP32_OPS_REFERENCE_FILE = Path(__file__).parent.resolve() / Path("configs/fp32_ops.txt")
```

Comment:
We still need those in `test_write_bf16_fp32_ops_to_text_files`

Reply:
done